crypto/conf/conf_mod.c

```diff
@@ -168,11 +172,11 @@ int CONF_modules_load_file_with_libctx(OPENSSL_CTX *libctx,
     if (NCONF_load(conf, file, NULL) <= 0) {
         if ((flags & CONF_MFLAGS_IGNORE_MISSING_FILE) &&
             (ERR_GET_REASON(ERR_peek_last_error()) == CONF_R_NO_SUCH_FILE)) {
-            ERR_clear_error();
             ret = 1;
         }
         goto err;
     }
+    ERR_pop_to_mark();
```

Comment:
This one seems too much, considering they are popped or cleared after `err:` as well...

Reply:
Ah, right! Removed.